client/extensions/woocommerce/app/orders/orders-list.js

```diff
@@ -183,11 +203,11 @@ class Orders extends Component {
 }
 
 export default connect(
-	state => {
+	( state, props ) => {
```

Comment:
Optional suggestion - pull this out into a mapstatetoprops method like we do elsewhere in the extension